traiders/backend/api/serializers/portfolio.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import serializers
+from ..models import Portfolio
+from django.contrib.auth.models import AnonymousUser
+
+
+class IsFollowingField(serializers.BooleanField):
+    def get_attribute(self, instance: Portfolio):
+        return instance.followed_by.filter(pk=self.context['request'].user.pk).exists()
+
+
+class PortfolioSerializer(serializers.HyperlinkedModelSerializer):
+    is_following = IsFollowingField()
+
+    class Meta:
+        model = Portfolio
+        fields = ["name", "owner", "followed_by", "parities"]
+
+    def update(self, instance: Portfolio, validated_data: dict):
+        is_following = validated_data.pop('is_following', None)
```

Comment:
TODO
1 - Check whether the owner of the portfolio is trying to follow their own portfolio. Raise permission error.
2 - Check whether some user other than the owner trying the update a field other than `is_following`. i.e: another user cant change the `name`. Raise permission error.

Reply:
done :+1: